pkg/code/utils.go

```diff
@@ -33,24 +33,24 @@ func CreateCodeIntegration(ctx context.Context, codeIntegrations []CodeIntegrati
 	return AddCodeIntegration(ctx, name)
 }
 
-func GetCodeIntegrationFromFlag(ctx context.Context, codeIntegrationIdFlag string, askForNewProjectFirst bool) (*CodeIntegration, error) {
+func GetCodeIntegrationFromFlag(ctx context.Context, codeIntegrationIdFlag string, askForNewProjectFirst bool) (code *CodeIntegration, isCreatedNew bool, err error) {
```

Comment:
```suggestion
func GetCodeIntegrationFromFlag(ctx context.Context, codeIntegrationIdFlag string, askForNewProjectFirst bool) (code *CodeIntegration, createNewCodeIntegration bool, err error) {
```
Best to keep names to make it easy to follow